app/src/backend/mocks/createCollection.ts

```diff
@@ -0,0 +1,55 @@
+import firebaseApp from "../../firebase";
+import { getFirestore, Timestamp, updateDoc, addDoc, collection } from "firebase/firestore";
+import { getOwnerId } from "backend/utils";
```

Comment:
<img width="1206" alt="CleanShot 2024-05-04 at 13 33 22@2x" src="https://github.com/requestly/requestly/assets/16779465/c5f0f552-da8a-42ad-a84a-9ffa1e3cff4d">

- Remove shown on non grouped mocks too.
- Remove is really vague. Can we rename it to `Remove from Collection` instead or just  add support for Ungroup in the Move modal
- Rename Copy to Copy Url instead

Reply:
`Remove shown on non grouped mocks too.` It shows, there but its disabled

Reply:
This is fixed in bulk action PR